tutorials/javaInstallationLinux.md

```diff
@@ -0,0 +1,64 @@
+{% set title="Java 17 Installation Guide for Linux Users" %}
+<frontmatter>
+  title: "{{ title }}"
+  pageNav: 2
+</frontmatter>
+
+# {{ title }}
+
+This guide explains how to install the **==[OpenJDK Java 17](https://openjdk-sources.osci.io/openjdk17/)==** on Linux using the command line. This guide was created using Ubuntu 22.04 but the steps for other Debian based Linux distros should be similar.
+
+
+## Installation
+This should be the default option for users with admin access to their system.
```

Comment:
Since this is worded as the default option, are there other options for users (both with admin access as well as maybe those without)?

Reply:
@mfjkri it is possible to install java without admin access, but I felt that putting that option in the guide would be confusing for new users, so I omitted it. I wanted to emphasis here that these steps require extra privileges. Maybe a better way to phrase this would be to replace the whole line with "This guide assumes you have super user access."?

Reply:
Thanks for the explanation! I agree, mentioning non-admin installations can complicate things for new users. The revised wording should work well.

Reply:
@mfjkri great, I've included both this and the link change in the latest few commits